src/main/resources/scipooldev.properties

```diff
@@ -4,6 +4,7 @@ AWS_REGION=us-east-1
 SESSION_TIMEOUT_SECONDS=28800
 SESSION_NAME_CLAIMS=userid
 SESSION_TAG_CLAIMS=sub,userid,team,user_name,company,given_name,family_name
+REDIRECT_URIS=https://synapse-login-scipoolprod.scipoolprod.org/synapse,https://connect.scipooldev.org/oauth2/idpresponse
```

Comment:
i believe this should be referencing scipooldev, `https://synapse-login-scipooldev.scipooldev.org/synapse`